airflow/providers/amazon/aws/sensors/redshift.py

```diff
@@ -17,12 +17,12 @@
 # under the License.
 import warnings
```

Comment:
Since `airflow.providers.amazon.aws.sensors.redshift` is a file that already released (long ago actually) We can't do here what I suggested for `redshift_cluster` in https://github.com/apache/airflow/pull/20374#discussion_r771535786

Here we need to make sure that both `AwsRedshiftClusterSensor` and `RedshiftClusterSensor` are working to the users.